tests/test_utils.py

```diff
@@ -94,3 +94,27 @@ def __init__(self, force=False, verbose=0):
     )
     assert test.force
     assert test.verbose == 2
+
+
+def test_get_dict_value_from_path_should_return_given_value():
+    """The get_dict_value_from_path function should return the value when it's present."""
+
+    dictionary = {"foo": {"bar": "bar_value"}}
+    path = ["foo", "bar"]
```

Comment:
To have a uniform test structure for `get_dict_value_from_path`, why don't parametrize this test for `path` values?

Reply:
Good Idea) Thanks) Indeed this will make it more uniform)